tasks/main.yml

```diff
@@ -26,6 +26,16 @@
   register: autoclean_output
   changed_when: "'Del' in autoclean_output.stdout"
 
+- name: APT | Check for cached .deb files
+  shell: "ls /var/cache/apt/archives/*.deb 2&> /dev/null | wc -l"
+  register: apt_cache_check
+  changed_when: apt_cache_check.stdout != "0"
+  when: apt_clean
+
+- name: APT | Remove all cached .deb files
```

Comment:
It might make more sense to just have one block like:

```yaml
- name: APT | Remove all cached .deb files
  shell: apt-get -y clean && find /var/cache/apt/archives/ -name *.deb | wc -l
  when: apt_clean
  register: deb_count
  changed_when: deb_count.stdout != "0"
```
What do you think?